server/src/main/java/server/models/User.java

```diff
@@ -20,6 +20,14 @@ public class User {
     @Column(name = "username")
     private String username;
 
+    @NotBlank
```

Comment:
Ha ezeket is későbbiekben kell állítani mint a profilképet (szerintem), akkor ezek is blank-ként kezdenek.
De nem fogok hazudni kb fölös h itt vannak mert nem is használjuk az ellenőrzést még lol